packages/components/tooltip/src/use-tooltip.ts

```diff
@@ -63,6 +68,11 @@ interface Props extends Omit<HTMLNextUIProps, "content"> {
    * @default document.body
    */
   portalContainer?: Element;
+  /**
+   * Whether the scroll should be blocked when the tooltip is open.
+   * @default true
+   */
+  shouldBlockScroll?: boolean;
```

Comment:
I think it's unnatural for a tooltip element that appears on hover to block scrolling.
Wouldn't it be better to prevent it from receiving props?